docs/contributing/Contributing-To-Ignite.md

```diff
@@ -22,6 +24,39 @@ node your-ignite-repo-clone/bin/ignite generate component OrderButton
 
 In addition, we have automated tests you can run with `yarn test`.
 
+### Making PRs
+
+We appreciate your contribution! To make it easier for us to review, please make sure to add a clear title and description that explains both what the change is, and why it's useful, if applicable.
+
+Maintainers may choose to rewrite this description for our changelog, or squash your commits when merging.
+
+## For maintainers
+
+### Merging PRs
+
+When merging PRs, we need maintainers to make sure that there's appropriately formatted commit message(s) for semantic-release to document changes for the release notes and pick the next-released version for Ignite. See [Releasing Ignite](./Releasing-Ignite.md) for more info on how to format commit messages.
+
+You have two choices when merging to Ignite:
+
+1. Merge with a merge commit, keeping the original commits. In this case, if the original commits are using the semantic-release conventions, those messages will be used for the changelog and release determinations.
+   1. If everything in the PR is sufficiently documented by those commit messages, you don't need to add anything to the commit message.
+   1. If there are no change entries, you should update the merge commit message when you merge to match the expected conventions.
+1. Alternatively, if you don't want to keep the original commits, you can use a squash merge. You will need to update the merge commit message to match semantic-release conventions as required by the particular change.
+
+Not every included commit in a PR needs to be formatted this way, only enough commits to ensure relevant changes have been documented.
```

Comment:
There are some examples of commit messages and semver if it makes sense to link them somewhere around here at https://docs.infinite.red/ignite-cli/contributing/Releasing-Ignite/#a-note-about-semver

Also for folks merging, adding the `(#PRNUMBER by @gituser)` in the merge/squash commit is helpful for the changelog (also pictured in the examples but not necessarily called out)

Reply:
@frankcalise I did link that, do we want to do that differently?

> See [Releasing Ignite](./Releasing-Ignite.md) for more info on how to format commit messages.

I can call out the contributor shout-out specifically here as well.

Reply:
@lindboe all good I had missed the link getting fixated on the markdown numbered list thing 😆 . But yeah the contributor piece is a good touch I think!

Reply:
Updated in [35558cc](https://github.com/infinitered/ignite/pull/2781/commits/35558ccbc2aae4808c81e56fb040cbfeb01d6b54)